e2e/suites/management/create_new_scheduler_version_test.go

```diff
@@ -91,7 +91,7 @@ func TestCreateNewSchedulerVersion(t *testing.T) {
 
 			err = managementApiClient.Do("GET", fmt.Sprintf("/schedulers/%s", scheduler.Name), getSchedulerRequest, getSchedulerResponse)
 			require.NoError(t, err)
-			require.Equal(t, "v1.0.0", getSchedulerResponse.Scheduler.Spec.Version)
+			require.Equal(t, "v1.1.0", getSchedulerResponse.Scheduler.Spec.Version)
```

Comment:
Why is this version  v1.1.0? Since we just created the scheduler and the create new version failed, it should keep on version v1.0.0, no?

Reply:
Self-answered in my last comment
I would recommend making it more clear that the scheduler returned has the version v1.1.0, but the way it is, works!